aiidalab_widgets_base/__init__.py

```diff
@@ -1,20 +1,19 @@
 """Reusable widgets for AiiDA lab applications."""
-# pylint: disable=unused-import
+# pylint: disable=unused-import,wrong-import-position
 from aiida import load_profile
 load_profile()
```

Comment:
I assume the 'wrong-import-position' is caused by this intermittent call? Could we just move it after the imports or disable the warning only for this line?

Reply:
No, otherwise we won't be able to load AiiDA objects

Reply:
But I can disable the warning for this line, this is true.

Reply:
Nope, didn't work. Prospector starts to complain about every subsequent import line.

Reply:
Ok, then let's go with the module-wide override.